source/materials/XenonLiquidProperties.cc

```diff
@@ -124,5 +124,13 @@ namespace nexus {
 
 
 
+  void XenonLiquidProperties::Scintillation
+  (std::vector<G4double>& energy, std::vector<G4double>& intensity)
+  {
+    for (G4int i=0; i<energy.size(); i++)
```

Comment:
When compiling this warning shows up in both `XenonLiquidProperties.cc` and `XenonGasProperties.cc`:

`warning: comparison of integers of different signs: 'G4int' (aka 'int') and 'std::__1::vector<double, std::__1::allocator<double> >::size_type' (aka 'unsigned long') [-Wsign-compare]`

Reply:
Replace `G4int` with `unsigned`.